app/lib/report_helper.rb

```diff
@@ -33,53 +33,65 @@ def compute_taxon_zscores(report)
     zscore_array
   end
 
-  def taxonomy_details(view_level, _report, params, taxon_zscores)
-    sz = select_zscore(view_level, taxon_zscores)
+  # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
ditto with these -- do we actually want them applied elsewhere?

Reply:
my personal preference is to not have a bunch of rubocop exception comments littering our code

Reply:
Yes I think these warnings are less useful.   They tend to steer us toward making the code more compact, but sometimes that makes it more difficult to understand.